README.md

```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
 * [checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html)
 * [Google java format](https://github.com/google/google-java-format)
 
+* In order to setup this project it's necessary:
```

Comment:
`it's necessary to:`

Also make sure each point makes sense, they all should start with a verb

Reply:
Done.